playground-common/playground.properties

```diff
@@ -29,3 +29,4 @@ androidx.playground.snapshotBuildId=8498920
 androidx.playground.metalavaBuildId=8444773
 androidx.playground.dokkaBuildId=7472101
 androidx.studio.type=playground
+kotlin.mpp.enableCInteropCommonization=true
```

Comment:
Should this be in `playground-common/androidx-shared.properties` + regulard androidx `gradle.properties`? I realize we may also need to update build warning baseline and `ValidatePlaygroundGradleProperties` task if we do